test/discljord/connections_test.clj

```diff
@@ -77,7 +77,7 @@
                                                 "t" "READY"
                                                 "d" {"session_id" "session"}})))
                         nil)))]
-      (fake/with-fake-http ["https://discord.com/api/gateway/bot?v=8&encoding=json"
+      (fake/with-fake-http ["https://discord.com/api/gateway/bot?v=9&encoding=json"
```

Comment:
Could you change this to use the gateway api version var from discljord.http?

Reply:
Done.